gaps/gap-6_tech_council_process/gap-6_tech_council_process.md

```diff
@@ -0,0 +1,79 @@
+---
+gap: 6
+title: Tech Council process
+description: Rules of operation for Golem Tech Council
+author: stranger80 (@stranger80)
+status: Draft
+type: Meta
+---
+
+## Abstract
+This is proposal to structure the proceedings of Golem Tech Council in order to streamline the flow of agenda items and make the approval process efficient.
+
+## Motivation
+The purpose is to ensure the Tech Council operates efficiently, with as few synchronous actions (like meetings) as possible, and with transparency to all Golem Factory team members.
+
+## Specification
+
+The main purpose of this proposed process is to make proceedings of Tech Council items more efficient and asynchronous (eg. have them raised and reviewed offline, and only use Tech Council sessions to discuss questions on items which have been already digested by participants). 
+
+This is why responsibilities of the Facilitator also must include upfront facilitation of Tech Council session agenda, so that the participants aregiven an early heads-up and are able to prepare for the session (eg. by reading and reviewing items proposed on the agenda).
```

Comment:
just spelling
```suggestion
This is why the responsibilities of the Facilitator also must include upfront facilitation of the Tech Council session agenda so that the participants are given an early heads-up and are able to prepare for the session (eg. by reading and reviewing items proposed on the agenda).
```